uberenv.py

```diff
@@ -195,6 +195,13 @@ def parse_args():
                            " but can cause issues with macOS versions >= 10.13. "
                            " so it is disabled by default.")
 
+    # option to stop after spack config
+    parser.add_option("--spack-only",
```

Comment:
Though maybe a different name for this, though the help comment helps.  Maybe clarifying that to be:

Download and configure Spack only.  No further spack commands will be called.

Reply:
Improved!